python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -373,8 +448,7 @@ async def take_electrometer_scan(
 
         electrometer_exposures = list()
 
-        while not exposures_done.done():
-
+        if not math.isnan(exposure_time):
```

Comment:
Thought about using None, but went with np.nan because there is some code that make it float(exposure_time) and it's unhappy. I'll get rid of `math` though!

Reply:
You can make the code that uses `float(exposure_time)` happy with something like:

```
float(exposure_time) if exposure_time is not None else None
```

This will probably also require changing some type annotations to make sure whatever is receiving the float, also accepts a `None`, e.g.:

```
    async def _take_data(
        ...
        fiber_spectrum_exposure_time: float | None,
        electrometer_exposure_time: float | None,
    ) -> dict:
```

Reply:
Do you feel strongly about not using np.nan? I can make these changes but they aren't necessary if we use np.nan